### PR TITLE
APPS-3555 add mapper

### PR DESCRIPTION
**Pull request by sergeyspryker**

### CODE REVIEW

File: config/spryk/templates/Zed/Persistence/Mapper/ZedPersistenceMapper.php.twig [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{
+    /**
+     * @param \Generated\Shared\Transfer\{{ domainEntity }}CollectionTransfer ${{ domainEntity | lcfirst }}CollectionTransfer
+     *
+     * @return \Orm\Zed\{{ module }}\Persistence\Base\{{ tableName | camelCased }}[]|\Propel\Runtime\Collection\ObjectCollection
+     */
+    public function map{{ domainEntity }}TransferCollectionTo{{ domainEntity }}EntityCollection(
+        {{ domainEntity }}CollectionTransfer ${{ domainEntity | lcfirst }}CollectionTransfer
+    ): ObjectCollection {
+        ${{ domainEntity | lcfirst }}EntityCollection = new ObjectCollection();
```
Comment (glazin-logema) [inline config/spryk/templates/Zed/Persistence/Mapper/ZedPersistenceMapper.php.twig:22]: Usually we use the object we want to fill in the result as the second argument of the method, wouldn't it be more correct to do the same here?

Comment (sergeyspryker) [reply config/spryk/templates/Zed/Persistence/Mapper/ZedPersistenceMapper.php.twig:22]: I've just taken an example from
\Spryker\Zed\CompanyUser\Persistence\Mapper\CompanyUserMapper. I'll try to find out

**Review by stereomon — CHANGES_REQUESTED** [review]
Please use the provided examples from the TechSpec https://spryker.atlassian.net/wiki/spaces/AOP/pages/3224371207/WIP+APPS-3483+CRUD+Facade+function+RFC+to+Spryk#Add-Mapper

**Review by stereomon — APPROVED** [review]
Please resolve conflicts and merge

**Pull request merged**